source/includes/tutorial1/_inkling.html.md

```diff
@@ -5,131 +5,96 @@
+}
 ```
 
-A `concept` in Inkling defines what you are going to teach the AI. By declaring a concept, you are instructing the BRAIN server that this a part of the BRAIN's _mental model_ that must be learned. Consequently, concept nodes must have corresponding curricula to teach them.
+A `concept` in Inkling defines what you are going to teach the AI. By declaring a concept, you are instructing the AI Engine to learn a new function that maps a set of inputs to outputs. Each concept must include a curriculum.
 
-In this simulation we are asking the agent to learn the concept `find_the_target` by choosing a direction (`predicts (PlayerMove)`) after seeing the current state (`follows input(GameState)`) of the simulation.
+In this simulation, we are asking the agent to learn the concept `FindTheTarget` by choosing a direction (of type `PlayerMove`) after seeing the current state (of type `GameState`).
 
 For more information about using the concept keyword, refer to the [Concept Reference][3].
 
 ## Curriculum and Lessons
 
-```inkling
-curriculum learn_curriculum
-    # this is the name of our concept from above
-    train find_the_target
-    # this is our simulator name
-    with simulator move_a_point_sim
-    # This is the name of our objective function in the simulator
-    objective reward_shaped
-        lesson get_close 
-            configure
-                constrain dummy with Int8{-1}
-            until
-                # This is again the name of the objective function
-                maximize reward_shaped
-end
+```inkling2
+curriculum {
+    # Specify a simulator as the data source
+    source MoveAPointSim
+}
 ```
 
-A `curriculum` in Inkling is used to define what and how to teach a concept. Each concept needs a corresponding curriculum to teach it. A `lesson` is part of a curriculum; it teaches a specific piece of the concept by specifying training and testing parameters and a stopping point (`until`). Lessons enable the BRAIN to learn the concept bit-by-bit instead of all at once (using multiple lessons will be covered in a future tutorial). Lessons are contained within curriculum statements. Each curriculum must contain at least one lesson.
-
-A curriculum contains information the Bonsai AI Engine uses to train your BRAIN on the concept you've specified. They also specify the reward function (`objective`) for teaching a given concept. The reward function is a way the system concretely measures the AI's performance as it learns the concept. For this tutorial we will only using one concept, and therefore, one curriculum. 
-
-Every lesson must have a configuration. In this case, we have a simple example that doesn’t need a configuration, so the config schema is named `dummy` with a default value of `-1` which in this case does nothing to configure the lesson.
-
-For more information about using these keywords, refer to the [Curriculums Reference][4] and [Lessons Reference][5].
-
-# The Inkling/Simulation Relationship
```

Comment:
This section was added based on common confusion about how all the pieces fit together. It's a shame to lose it, at least until we simplify things so that it's obvious.